helper.php

```diff
@@ -420,10 +536,16 @@ function _diffCell($id) {
 
     /**
      * Plugins - respective plugins must be installed!
+     * @see addColumn for more information
      */
     function _pluginCell($plug, $col, $id) {
-        if (!isset($this->page[$col])) $this->page[$col] = $this->$plug->td($id);
-        return $this->_printCell($col, $this->page[$col]);
+        // At this point we do not need to differ between built-in and other plugins
+        $class = '';
+        if (!isset($this->page['plugins'][$plug][$col])) $this->page['plugins'][$plug][$col] = $this->$plug->td($id, $col, $class);
```

Comment:
Please use brackets

Reply:
Added missing brackets to all if-statements.